emeritus-tech/landing-pages-libraries
# [CPT-518] Columbia: VI-PM markup and stylesheets

🎟 [Jira Ticket CPT-518](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-518) 
🎟 [Jira Ticket CPT-519 ](https://emeritus.atlassian.net/secure/RapidBoard.jspa?rapidView=24&projectKey=CPT&modal=detail&selectedIssue=CPT-519)

🚢 Deployment Instructions

N/A

✅ QA instructions

_** All modals should work as expected (without the modals.js file will only function in fallback mode, meaning that only clicking the X button will close it and ESC key or clicking outside will not)_

_** Testimonial slider should work as expected using CSS only (not entirely the same as the original landing page one)_

To test **Columbia Value Investing**:

*  Run command `npm start`
*  Open `http://localhost:8080/CVI.html`
*  And verify on browser:
   *  Design should look exactly as original states [landing CP](https://online1.gsb.columbia.edu/value-investing?ab_test[e20191209_static_landing_pages_course_preview]=variant), [landing legacy](https://online1.gsb.columbia.edu/value-investing?ab_test[e20191209_static_landing_pages_course_preview]=control), [thank you]( https://online1.gsb.columbia.edu/value-investing/thank-you?country_code=US) and [apply.](https://emeritus.gsb.columbia.edu/value-investing-online/apply.php)

To test **Columbia Python for Managers**:

*  Run command `npm start`
*  Open `http://localhost:8080/CPM.html`
*  And verify on browser:
   *  Design should look exactly as original states [landing CP](https://online1.gsb.columbia.edu/columbia/python-for-managers?ab_test[e20191209_static_landing_pages_course_preview]=variant), [landing legacy](https://online1.gsb.columbia.edu/columbia/python-for-managers?ab_test[e20191209_static_landing_pages_course_preview]=control), [thank you](  https://online1.gsb.columbia.edu/columbia/python-for-managers/thank-you) and [apply.]( https://emeritus.gsb.columbia.edu/python-for-managers/apply.php)

🖼 Previews

**Columbia Value Investing**

![screencapture-localhost-8080-CVI-html-2020-05-13-09_44_41](https://user-images.githubusercontent.com/58177278/81813903-6aa4a300-94fe-11ea-86f7-40205f4b5658.png)

**Columbia Python for Managers**

![screencapture-localhost-8080-CPM-html-2020-05-15-14_40_13](https://user-images.githubusercontent.com/58177278/82080020-35dd4b00-96ba-11ea-879f-f163668c6148.png)
